style.css

```diff
@@ -110,12 +110,9 @@ header {
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 114px 24px;
+  padding-top: 88px;
   gap: 88px;
-  position: absolute;
-  width: 375px;
-  height: 2788px;
-  top: 718px;
+  height: 100%;
 }
 
 .works {
```

Comment:
- [OPTIONAL ] Kindly make sure you use the proper paddings and margins for your cards to make it look like the Figma design :recycle: 

Your design:
![Screenshot from 2023-01-27 14-12-54](https://user-images.githubusercontent.com/97834160/215084854-6a4848b5-4cdd-4c74-8407-ceb933d37bb3.png)
- Figma:
![Screenshot from 2023-01-27 14-12-19](https://user-images.githubusercontent.com/97834160/215084860-19c28219-4dcf-45e1-923a-180a92a86589.png)